src/Symfony/Bundle/FrameworkBundle/Translation/PhpExtractor.php

```diff
@@ -20,7 +21,7 @@
  *
  * @author Michel Salib <michelsalib@hotmail.com>
  */
-class PhpExtractor implements ExtractorInterface
+class PhpExtractor extends AbstractFileExtractor implements ExtractorInterface
```

Comment:
I thought about this too but decided to go this way as the abstract class doesn't implement any of the methods from the interface.